### DISCUSSION TITLE
[UI] Pause Syncs Button

### DISCUSSION THREAD

**Discussion by justin-autobinck** [airbytehq/airbyte#32739] [Connector Ideas and Features]
### What area the feature impact?

Airbyte Platform

### Relevant Information

I have been searching for a good answer to this but it doesn't seem possible. 

I often want to pause the syncing job of a connection (mid job) for a day or two (due to too many and various reasons to get into now). 

A nice big orange button to pause the current job would be great, though I can only imagine that such a feature would take quite the effort to implement. If I am missing some alternative way to achieve the same thing please let me know.

**Comment by nataliekwong** [comment]
Hey @justin-autobinck , there is no explicit way to pause a sync today.

To better gauge how to prioritize work on this feature, could you describe your use case of what you're trying to accomplish and why?

**Reply by justin-autobinck** [reply]
@nataliekwong sure!

At one of the organisations I work they have a local MongoDB instance used mostly for operational data. For a rather small setup there are often large volumes of data entering/exiting this db. This also happens at different times of the day so it is a bit too tricky to just get a good schedule as to when to sync data. 

The problem is that the engineering teams want full IO for their jobs when large dumps are happening, and that means me stopping the sync in Airbyte. 

Is this too much of an edge case for implementing a whole new feature?

[a comment by an automated account is not shown]
